front/extension/app/src/pages/MainPage.tsx

```diff
@@ -1,12 +1,18 @@
 import { GenerationContextProvider } from "@app/components/assistant/conversation/GenerationContextProvider";
 import { FixedAssistantInputBar } from "@app/components/assistant/conversation/input_bar/InputBar";
+import { useAuth } from "@app/extension/app/src/context/AuthProvider";
 import { Page } from "@dust-tt/sparkle";
 import type { WorkspaceType } from "@dust-tt/types";
 
 export const MainPage = () => {
+  const { token } = useAuth();
+  if (!token) {
+    return <div>Not logged in!!!</div>;
+  }
+
   const owner: WorkspaceType = {
     id: 1,
-    sId: "IQw2NP0Anb",
+    sId: "7ea8c3d99c",
```

Comment:
This will go away as soon as we've merged https://github.com/dust-tt/dust/pull/8085 and I can replace with a call from /me